meson.build

```diff
@@ -0,0 +1,22 @@
+# #############################################################################
+# Copyright (c) 2018-present    lzutao <taolzu(at)gmail.com>
+# All rights reserved.
+#
+# This source code is licensed under both the BSD-style license (found in the
+# LICENSE.meson-wrap-from-upstream.build file in the root directory of this
+# source tree) and the GPLv2 (found in the
+# COPYING.meson-wrap-from-upstream.build file in the root directory of this
+# source tree).
+# #############################################################################
+
+# This is the dummy meson file from upstream, with version and path adjusted.
+
+project('lz4', ['c'],
+  license: ['BSD', 'GPLv2'],
+  default_options : ['c_std=c99', 'buildtype=release'],
+  version: '1.9.3',
+  meson_version: '>=0.47.0')
+
+subdir('contrib/meson/meson')
+
+lz4_dep = liblz4_dep # For consistency with the wrap name
```

Comment:
If using [provide], do you care about the wrap name? And the pkg-config name is liblz4...